shell/platform/android/io/flutter/view/FlutterNativeView.java

```diff
@@ -112,7 +121,7 @@ public boolean isApplicationRunning() {
     }
 
     public static String getObservatoryUri() {
-        return nativeGetObservatoryUri();
+        return new FlutterJNI().nativeGetObservatoryUri();
```

Comment:
Curious: is it possible to make that method static?

Reply:
Upon further reflection, I think we should actually make FlutterJNI more stateful.  Currently FlutterJNI is exposing the native ID and then FlutterJNI asks for it back in all of its methods.  It would make more sense to keep that info within FlutterJNI.  So tomorrow I'm going to make that refactor and see how it looks.  Then it will make sense that none of the native methods are static, and the registration methods will make more sense, too.